packages/roosterjs-editor-core/lib/editor/createEditorCore.ts

```diff
@@ -52,10 +54,24 @@ export default function createEditorCore(
         eventHandlerPlugins: eventHandlerPlugins,
         api: createCoreApiMap(options.coreApiOverride),
         defaultApi: createCoreApiMap(),
+        inDarkMode: darkModeOptions ? true : false,
+        darkModeOptions: darkModeOptions,
     };
 }
 
-function calcDefaultFormat(node: Node, baseFormat: DefaultFormat): DefaultFormat {
+function calcDefaultFormat(node: Node, options: EditorOptions, darkModeOptions?: DarkModeOptions): DefaultFormat {
+    let baseFormat = null;
+    if (darkModeOptions) {
+        baseFormat = darkModeOptions.defaultFormat ? darkModeOptions.defaultFormat : <DefaultFormat>{

```

Comment:
This probably furthers the idea there should be a const file somewhere that can be used to store the default dark mode formatting.

Reply:
👍 for adding a const for that